src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -142,6 +142,18 @@ public static String sanitizeForRichText(String content) {
         }
         return policy.sanitize(content);
     }
+    
+    /**
+     * Sanitizes the {@link com.google.appengine.api.datastore.Text} with rich-text.
+     * Removes disallowed elements based on defined policy.
+     * @return A new sanitized {@link com.google.appengine.api.datastore.Text} or null if the input was null.
```

Comment:
Is it necessary to use fully qualified name? Unless there are name collisions, using just the class name would work just fine.